packages/web/src/scss/components/Pagination/README.md

```diff
@@ -0,0 +1,181 @@
+# Pagination
+
+First item selected:
+
+```html
+<nav role="navigation" aria-label="Pagination Navigation" class="Pagination mb-800">
+  <ul class="Pagination__items">
+    <li>
+      <a href="#" class="Button Button--secondary Button--square Button--disabled">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#chevron-left"></use>
+        </svg>
+        <span class="accessibility-hidden">Previous</span>
+      </a>
+    </li>
+    <li>
+      <a
+        href="#"
+        class="Pagination__item Pagination__item--selected"
```

Comment:
I would prefer `--current`
1. it would be consistent with the Header
2. it would be consistent with ARIA attribute

Reply:
Yes, that is more consistent.